src/components/reusable/svgSprite/SvgSprite.tsx

```diff
@@ -0,0 +1,23 @@
+import { FC, JSX } from 'react'
+
+interface TProps {
+    id: string,
+    size?: [number|string, number|string],
```

Comment:
can these params be required?

Reply:
can, but the idea for svgSprite was that if you don't set these values, the default values will be applied. I didn't want the component to have a lot of input attributes, which in fact can not be used. Do you think it makes sense?